models/pattern/core/pattern.go

```diff
@@ -199,7 +199,7 @@ func NewPatternFileFromCytoscapeJSJSON(byt []byte) (Pattern, error) {
 
 	// Convert cytoscape struct to patternfile
 	pf := Pattern{
-		Name:     "MesheryGeneratedPatternFile",
+		Name:     name,
```

Comment:
Don't we need a fallback in case the name is missed?

Reply:
Yes, we do.